config/v1/types_project.go

```diff
@@ -19,9 +18,20 @@ type Project struct {
 	Status ProjectStatus `json:"status"`
 }
 
+// TemplateReference references a project request template in a 'openshift-config' namespace.
+type TemplateReference struct {
+	// name is the metadata.name of the referenced project request template
+	Name string `json:"name"`
+}
+
 type ProjectSpec struct {
-	// project request message
-	// project request template
+	// projectRequestMessage is the string presented to a user if they are unable to request a project via the projectrequest api endpoint
+	ProjectRequestMessage string `json:"projectRequestMessage"`
+
+	// projectRequestTemplate is the template to use for creating projects in response to projectrequest.
+	// This must point to a template in 'openshift-config' namespace. It is optional.
+	// If it is not specified, a default template is used.
+	ProjectRequestTemplate *TemplateReference `json:"projectRequestTemplate"`
```

Comment:
@deads2k @mfojtik this should not be a pointer.

Reply:
why? the field is optional... actually I think it is missing omitempty flag.

Reply:
> @deads2k @mfojtik this should not be a pointer.

True, I missed that.  There is no functional difference between empty and nil